whitespace_work_title_cleanup_bot/whitespace_work_title_cleanup.py

```diff
@@ -39,12 +38,12 @@ def run(self) -> None:  # overwrite the AbstractBotJob run method
                 old_title = copy.deepcopy(work.title)
                 work.title = work.title.strip()
                 self.logger.info(
-                    '|'.join([olid, old_title, work.title])
+                    "|".join([olid, old_title, work.title])
                 )  # don't forget to log modifications!
                 self.save(lambda: work.save(comment=comment))
 
 
-if '__name__' == '__name__':
+if "__name__" == "__main__":
```

Comment:
```suggestion
if __name__ == "__main__”:
```
Dunder name is a builtin variable in Python.